app/page.tsx

```diff
@@ -1,7 +1,15 @@
+"use client";
+
+import SearchBar from "@/components/SearchBar";
+import React from "react";
+
 export default function Home() {
+  const [albumSearch, setAlbumSearch] = React.useState<string>("");
```

Comment:
I think the naming could be better here:

```suggestion
  const [searchValue, setSearchValue] = React.useState<string>("");
```